x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/autoscaling/MlAutoscalingDeciderServiceTests.java

```diff
@@ -400,8 +400,10 @@ public void testScale_WithNoScaleUpButWaitingJobs() {
         DeciderContext deciderContext = new DeciderContext(clusterState, autoscalingCapacity);
 
         AutoscalingDeciderResult result = service.scale(settings, deciderContext);
-        assertThat(result.reason().summary(),
-            containsString("Passing currently perceived capacity as there are analytics and anomaly jobs in the queue"));
+        assertThat(
+            result.reason().summary(),
+            containsString("but the number in the queue is less than the configured maximum allowed")
```

Comment:
Is this the only message that needs changing?
How hard would it be to add a test for the revamped message starting with "Passing currently perceived capacity..."?

Reply:
The test would end up being a copy paste of the `String.format` because the input values are random. That seemed a bit ridiculous to me as it would basically be testing "Hey, does the production code act like code that is copy and pasted from it into a test?"